File: DAL/Models/UserLanguage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL.Models
{
    public class UserLanguage
    {
        public int Id { get; set; }
        public int LanguageId { get; set; }
        public Language Language { get; set; }
        public int EmployeeId { get; set; }
        public Employee Employee { get; set; }
        public int Level { get; set; }
    }
}

File: GlassDoor/ClientApp/src/app/models/user-language.ts
import { Employee } from "./employee";
import { Language } from "./language";

export class UserLanguage {

  constructor(id?: number, languageId?: number, language?: Language,
    employeeId?: number, employee?: Employee, level?: number,) {

    this.id = id;
    this.languageId = languageId;
    this.language = language;
    this.employeeId = employeeId;
    this.employee = employee;
    this.level = level;

  }
  public id: number;
  public languageId: number;
  public language: Language;
  public employeeId: number;
  public employee: Employee;
  public level: number;



}
